diff -ur Plack-0.9978-o/t/Plack-Middleware/conditionalget_writer.t Plack-0.9978/t/Plack-Middleware/conditionalget_writer.t
--- Plack-0.9978-o/t/Plack-Middleware/conditionalget_writer.t	2011-04-12 18:36:52.000000000 -0600
+++ Plack-0.9978/t/Plack-Middleware/conditionalget_writer.t	2011-05-10 22:14:27.000000000 -0600
@@ -5,6 +5,7 @@
 use HTTP::Request::Common;
 use Plack::Test;
 
+
 my $handler = builder {
     enable 'ConditionalGET';
 
@@ -26,12 +27,14 @@
     my $cb = shift;
 
     subtest 'streaming' => sub {
+	plan tests => 2;
         my $res = $cb->( GET "http://localhost/streaming-klingklangklong" );
         is $res->code, 200, 'Response HTTP status';
         is $res->content, 'klingklangklong', 'Response content';
     };
 
     subtest 'streaming not modified' => sub {
+	plan tests => 2;
         # the middleware does not support streaming interface but make it at least not die
         my $res = $cb->( GET
             "http://localhost/streaming-klingklangklong",
